### DISCUSSION TITLE
Where is source code of v compiler written in c ? (The very first version of Vlang)

### DISCUSSION THREAD

**Discussion by b00tkitism** [vlang/v#17418] [Questions and Answers] [answered]
.

**Comment by JalonSolov** [comment, marked as answer]
The very first version of V was written in Go, and will never be released.  The first public release of V was written in V - in other words, it could already compile itself.

**Reply by b00tkitism** [reply]
Correct, but why it will never be released ?

**Comment by medvednikov** [comment]
There's also this:

https://github.com/vlang/vc

V translated to a single v.c file for every commit.

**Reply by JalonSolov** [reply]
However, it isn't the first commit, it's the latest.  :-)

**Reply by medvednikov** [reply]
Ah, I missed the "very first version" part :)
